inst/scripts/SEL3D.R

```diff
@@ -0,0 +1,139 @@
+#Pseudocode may look like actual code...please ignore...
+getPdbIds <- function(targetSeq, startPos = 1 , endPos = nchar(targetSeq), convert = TRUE, silent = TRUE){
+  
+  #get packages
+
+  library(bio3d)
+  library(RCurl)
+  library(XML)
+  library(seqinr)
+  
```

Comment:
Please have a look at the following links
http://r-pkgs.had.co.nz/description.html
https://github.com/hyginn/rete/blob/master/DESCRIPTION